lib/OpenQA/WebSockets/Client.pm

```diff
@@ -21,37 +21,33 @@ use Carp 'croak';
 use OpenQA::Client;
 use OpenQA::Utils 'service_port';
 
-has host   => sub { $ENV{OPENQA_WEB_SOCKETS_HOST} };
-has client => sub { OpenQA::Client->new(api => shift->host // 'localhost') };
-has port   => sub { service_port('websocket') };
+has host   => sub ($self) { $ENV{OPENQA_WEB_SOCKETS_HOST} };
+has client => sub ($self, @) { OpenQA::Client->new(api => $self->host // 'localhost') };
```

Comment:
That's not an accessor, it's just a closure that lazily creates the initial value for the attribute. There should be no additional arguments getting passed to it.